Scripts/developer_scripts/list_cgal_workflows_last_run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+echo "  workflow  |  runs on  |  status of last run  |  date  |  file  |"
```

Comment:
Check the Markdown syntax:

It should be:
```suggestion
echo "| workflow  |  runs on  |  status of last run  |  date  |  file  |"
echo "| --------- | --------- | -------------------- | ------ | ------ |"
```

And remove the `echo "----------"` between lines.

Reply:
See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#creating-a-table